pkg/operator/ceph/cluster/osd/spec_test.go

```diff
@@ -29,15 +29,22 @@ import (
 	opconfig "github.com/rook/rook/pkg/operator/ceph/config"
 	operatortest "github.com/rook/rook/pkg/operator/ceph/test"
 	cephver "github.com/rook/rook/pkg/operator/ceph/version"
+	"github.com/rook/rook/pkg/operator/k8sutil"
 	exectest "github.com/rook/rook/pkg/util/exec/test"
 	"github.com/stretchr/testify/assert"
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/client-go/kubernetes/fake"
 )
 
 func TestPodContainer(t *testing.T) {
 	cluster := &Cluster{rookVersion: "23", clusterInfo: client.AdminClusterInfo("myosd")}
+	cephCluster := cephv1.CephCluster{}
```

Comment:
It seems adding this reference in the tests is a lot of duplicate code. Could we have a helper method to set it in the test? Or is there a pattern used in the other tests that could be followed here? I don't see the duplication in the other unit tests like this one.

Reply:
It's introduced in this PR. The original code just needs empty ownerReference but now one needs ownerInfo with setting some fields in many cases. I'll add some helper scripts.